doc/user/nhrpd.rst

```diff
@@ -189,11 +189,14 @@ nhrpd can be configured to forward multicast packets, allowing routing
 protocols that use multicast (such as OSPF) to be supported in the DMVPN
 network.
 
-This support requires an NFLOG redirection rule to work:
+This support requires an iptables NFLOG rule to allow nhrpd to intercept
+multicast packets. A second iptables rule is also usually used to drop the
+original multicast packet.
 
  .. code-block:: shell
 
-   iptables -I OUTPUT -d 224.0.0.0/24 -o gre1 -j NFLOG --nflog-group 2
+   iptables -A OUTPUT -d 224.0.0.0/24 -o gre1 -j NFLOG --nflog-group 2
+   iptables -A OUTPUT -d 224.0.0.0/24 -o gre1 -j DROP
 
```

Comment:
about this error message:
```
2021/03/17 08:48:32 OSPF: [EC 100663299] *** sendmsg in ospf_write failed to 224.0.0.5, id 46163, off 0, len 64, interface gre1, mtu 1452: Operation not permitted
```
it seems the error is derived from the DROP rule.
having flog_err messages should absolutely be avoided.
is it possible to detect errno "operation not permitted" to avoid that error message when sending ospf packets?

Reply:
I have put an extra change so that this failure message will be suppressed.